src/address/channel.rs

```diff
@@ -458,6 +460,10 @@ where
     fn hash(&self) -> usize {
         self.inner.as_ref() as *const _ as usize
     }
+
+    fn connected(&self) -> bool {
+        self.connected()
```

Comment:
Hmm just curious is compiler ok with this ambiguous method invocation?
Since both struct and trait has the same method it might cause problems

Reply:
Some similar methods 10 lines above (`do_send`/`try_send`/`send`) exists here a while.